.eslint-resolver.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+const assign = require ( 'lodash/assign' );
+const debugFactory = require( 'debug' );
+const nodeResolver = require( 'eslint-import-resolver-node' );
+const path = require( 'path' );
+const webpackResolver = require( 'eslint-import-resolver-webpack' );
+
+/**
+ * Internal dependencies
+ */
+const webpackResolveConfig = require( './webpack.config.resolve' );
+
+/**
+ * Module variables
+ */
+const log = debugFactory('eslint-plugin-import:resolver:wp-calypso-resolver');
+
+const nodeResolverConfig = {
```

Comment:
Can you explain the need for separate resolving configs? Should this be pulled from `webpack.config.node.js` for server-side module importing?